features/mesh/services/Item.feature

```diff
@@ -42,7 +42,7 @@ Feature: mesh / services / item
               - name: fake-dataplane
                 dataplane:
                   networking:
-                    gateway: ~
+                    gateway: !!js/undefined
```

Comment:
Did adding this make the tests fail? i.e. Now you've added this would it catch a similar issue in the future?

Reply:
Awesome thankyou!